CHANGELOG.md

```diff
@@ -1,228 +1,204 @@
-| Bug   | \[Issue] Improve Gallery/AddToCartButton.js                                                                                             | [3573][]          |
-| Bug   | \[bug]: Breadcrumbs 'Shop' link redirects to '/shopnull'                                                                                | [3603][]          |
-| Bug   | Newsletter form displays two errors                                                                                                     | [3584][]          |
-| Bug   | Home page tabbing leads to page transition                                                                                              | [3625][]          |
-| Bug   | Browser Reload on any Category load all products from default category                                                                  | [3655][], [18][]  |
+### Personalized content support
+
+-   Added shimmer loaders for Dynamic blocks to avoid layout shift. Shimmer loaders are only rendered if `minHeight` is specified on the Row containing the Dynamic block. This implementation gives you a choice: You can set the Row's `minHeight` to render the shimmer or remove the `minHeight` to remove the shimmer. For Dynamic blocks that rarely render, we recommend removing the `minHeight` property from the Rows containing those Dynamic blocks. — [3713][]
+-   Added the new `PersonalizedContentSampleDataVenia` module that contains customer segments, dynamic blocks, and cart rule promotion sample data for personalized content. — [19][],[2][],[3670][]
+-   Shopping cart promotions are now properly displayed depending on customer segment. — [3609][]
+-   You can now make GraphQL queries for dynamic blocks. — [3587][],[2][]
+
+### ReCaptcha support
+
+-   You can now use ReCaptcha V3 on the Braintree payment form and the Place Order form. — [3691][],[24][]
+-   reCAPTCHA V3 validation is now available on the following customer forms:
+    - Customer Login
+    - Customer account creation
+    - Customer information change (password) — [3702][]
+    - Forgot Password form. — [3604][],[12][],[20][]
+
+### PageBuilder content optimization
+
+-   Shimmers are now loaded when rendering a Page Builder CMS page. This helps reduce CLS and increase load speed. — [3711][]
+-   Images from Page Builder content now include height and width dimensions. This helps reduce CLS when rendering the images. — [3712][]
+-   Store-view specific label and localization support has been added for product attributes. — [17][]
+-   PWA Studio can now retrieve image metadata from Page Builder. — [16][],[17][]
+
+### Other highlights
+
+-  In the Link component, `prefetchType` property has been renamed to `shouldPrefetch`. This helps to clarify that it is a boolean, rather a property that returns a type. `prefetchType` is deprecated but not removed. — [3646][]
```

Comment:
Suggested re-write for clarity: 

"In the Link component, `prefetchType` property has been renamed to `shouldPrefetch`. This clarifies that this is a boolean, rather than a property that returns a type. `prefetchType` is deprecated but not removed."

Reply:
@ericeoeur Its just backmerge so this is not needed on this ticket.